tests/loader_layer_tests.cpp

```diff
@@ -783,17 +783,17 @@ TEST(OverrideMetaLayer, NewerComponentLayerInMetaLayer) {
         inst.create_info.set_api_version(1, 1, 0);
         inst.CheckCreate();
         VkPhysicalDevice phys_dev = inst.GetPhysDev();
-        ASSERT_TRUE(
-            env.debug_log.find("verify_meta_layer_component_layers: Meta-layer uses API version 1.1, but component layer 0 uses "
-                               "API version 1.2.  Skipping this layer."));
+        // Newer component is allowed now
+        EXPECT_TRUE(env.debug_log.find(std::string("Insert instance layer ") + regular_layer_name));
```

Comment:
ASSERT Stops the test right then right there, EXPECT logs an error but continues.
Generally we should be using expect and only ASSERT in places where we can't continue the test. Looking for log messages is something to EXPECT, while failing to create an instance is an ASSERT. However, its not too critical to use the 'wrong' one, since they will always cause the test to fail.